packages/harmony/src/hooks/useHotKeys.ts

```diff
@@ -32,7 +32,8 @@ function allowGlobalHotkeyPress() {
     (document.activeElement === document.body ||
       document.activeElement.nodeName === 'A' /* <a> */ ||
       document.activeElement.nodeName === 'BUTTON' /* <button> */ ||
-      document.activeElement.getAttribute('role') === 'button')
+      document.activeElement.getAttribute('role') === 'button' ||
+      document.activeElement.getAttribute('role') === 'main')
```

Comment:
I'm not entirely understanding this point of this hotkeypress check so I think this is fine? Definitely open to feedback here